packages/rancher-logging/generated-changes/overlay/templates/loggings/rke/logging-rke.yaml

```
@@ -1,4 +1,5 @@
 {{- if .Values.additionalLoggingSources.rke.enabled }}
+{{- $containers := printf "%s/%s" .Values.global.dockerRootDirectory "containers" -}}
```

The differing ways of doing this is odd. Why not same as above
```suggestion
{{- $containers := printf "%s/containers" .Values.global.dockerRootDirectory -}}
```
Also why not trailing slash here ? It might not make a difference but consistency would be good. I see same thing below also.

---

There's a few ways to go about this (https://github.com/rancher/charts/pull/995#discussion_r578102738), but I will make it consistent across the repository.